src/pipx/interpreter.py

```diff
@@ -41,27 +43,63 @@ def __init__(self, source: str, version: str, wrap_message: bool = True):
                 message += "The provided version looks like a path, but no executable was found there."
             if potentially_pylauncher:
                 message += (
-                    "The provided version looks like a version for Python Launcher, " "but `py` was not found on PATH."
+                    "The provided version looks like a version, "
+                    "but both the python command and the Python Launcher were not found on PATH."
                 )
         if source == "the python-build-standalone project":
             message += "listed in https://github.com/indygreg/python-build-standalone/releases/latest."
         super().__init__(message, wrap_message)
 
 
+def find_unix_command_python(python_version: str) -> Optional[str]:
+    try:
+        parsed_python_version = version.parse(python_version)
+    except version.InvalidVersion:
+        logger.info(f"Invalid Python version: {python_version}")
+        return None
+    else:
+        if (
+            parsed_python_version.epoch != 0
+            or parsed_python_version.is_devrelease
+            or parsed_python_version.is_postrelease
+            or parsed_python_version.is_prerelease
+        ):
+            logger.info(f"Unsupported Python version: {python_version}")
+            return None
+
+        # Python command could be `python3` or `python3.x` without micro version
+        python_command = f"python{'.'.join(python_version.split('.')[:2])}"
+
+        python_path = shutil.which(python_command)
+        if not python_path:
+            logger.info(f"Command `{python_command}` was not found on the system")
+            return None
+
+        if parsed_python_version.micro != 0:
+            logger.warning(
+                f"The command `{python_command}` located at `{python_path}` will be used. "
```

Comment:
Backticks are rather uncommon for command line output, I think.

```suggestion
                f"The command '{python_command}' located at '{python_path}' will be used. "
```